src/views/CompanyList.vue

```diff
@@ -114,6 +101,7 @@ export default {
       isLoading: true,
       searchQuery: '',
       initialQueryHashtags: [],
+      sortingOptions: ['claps', 'name', 'updated'],
```

Comment:
sortingOptions was being referenced below to set the default sorting on page load.
```
if (!this.$route.query.sort) {
      this.$router
        .replace({ query: { ...this.$route.query, sort: this.sortingOptions[2] } })
        .catch(() => {})
    }
```